pkg/naming/naming.go

```diff
@@ -1,3 +1,39 @@
 package naming
 
+import (
+	api "github.com/percona/percona-server-mongodb-operator/pkg/apis/psmdb/v1"
+)
+
 const perconaPrefix = "percona.com/"
+
+const (
+	FinalizerDeleteBackup           = perconaPrefix + "delete-backup"
+	FinalizerDeletePVC              = perconaPrefix + "delete-psmdb-pvc"
+	FinalizerDeletePSMDBPodsInOrder = perconaPrefix + "delete-psmdb-pods-in-order"
+)
+
+func GetOrderedFinalizers(cr *api.PerconaServerMongoDB) []string {
```

Comment:
I think this function should live in `finalizer.go`. Only the const names should be in this `naming.go`.